doc/landing/css/normalize.css

```diff
@@ -0,0 +1,527 @@
+/*! normalize.css v1.1.3 | MIT License | git.io/normalize */
```

Comment:
Where are all of these JS, CSS, PNG files coming from? Were they auto-generated by Doxygen, or were they manually brought in here?

Reply:
They are copies of the same files for the other languages, e.g.:

https://github.com/GoogleCloudPlatform/google-cloud-java/tree/gh-pages

Apparently it is what we do for other languages too:

https://github.com/GoogleCloudPlatform/google-cloud-go/commit/a8286b8b40cc15563164e833f43356778f5e35ff

I chose to keep the files in the `master` branch and have the CI scripts copy them to the right `gh-pages` subdirectory, because that gives us more visibility into their history and gets them reviewed.

Reply:
The general guidance is to put third-party content (which this is) into a `third_party` directory, rather than just putting it in something like `doc` on a branch.

Ideally, each project/source gets its own separate subdir of `third_party`, with a clear license, and a README that says where the content comes from (e.g., URL to project, home page, etc.).

The link you have to other repos point to their `gh-pages` branches which are derived/serving versions. I think there's more of an issue having them directly in `master` because that's a work product, not a compiled/derivative version.